qa/integration/README.md

```diff
@@ -0,0 +1,41 @@
+## Logstash Integration Tests aka RATS
+
+These set of tests are full integration tests as in: they can start LS from a binary, run configs using `-e` and can use any external services like Kafka, ES and S3. This framework is hybrid -- a combination of bash scripts (to mainly setup services), Ruby service files, and RSpec. All test assertions are done in RSpec.
+
+No VMs, all tests run locally.
+
+## Dependencies
+* An existing Logstash binary, defaults to `LS_HOME/build/logstash-<version>`
+* `rspec`
+
+## Preparing a test run
+
+1. If you already have a LS binary in `LS_HOME/build/logstash-<version>`, skip to step 5
+2. From Logstash git source directory or `LS_HOME` run `rake artifact:tar` to build a package
+2. Untar the newly built package
+3. `cd build`
+4. `tar xvf logstash-<version>.tar.gz`
+5. `cd LS_HOME/qa/integration`
+6. `bundle install`: This will install test dependency gems.
+
+You are now ready to run any tests from `qa/integration`.
+* Run all tests: `rspec specs/*`
+* Run single test: `rspec specs/es_output_how_spec.rb`
```

Comment:
Also, it would be great to add new shell scripts in the `ci` directory that will take care of doing all the necessary steps to run the tests. We could use that script on Travis/Jenkins.